tap_postgres/client.py

```diff
@@ -45,7 +48,26 @@ def patched_conform(
     """
     if isinstance(elem, datetime.date):
         return elem.isoformat()
-    return unpatched_conform(elem=elem, property_schema=property_schema)
+    if isinstance(elem, (datetime.datetime, pendulum.DateTime)):
+        return singer_sdk.helpers._typing.to_json_compatible(elem)
+    if isinstance(elem, datetime.timedelta):
```

Comment:
Could we call the super here instead of copying the code?

Reply:
Comments added explaining why this isn't possible.